packages/sdk/ios/Sources/Headless Universal Checkout/Managers/RNTRedirectManager.swift

```diff
@@ -0,0 +1,193 @@
+//
+//  RNTRedirectManager.swift
+//  example_0_70_6
+//
+//  Created by Faisal Iqbal on 15.12.2023.
+//
+
+import Foundation
+import PrimerSDK
+
+@objc
+enum RNTPrimerHeadlessUniversalCheckoutComponentWithRedirectManagerEvents: Int, CaseIterable {
```

Comment:
Would this work as a `String` enum? It would clean up the implementation a bit, and match Android

```
enum RNTPrimerHeadlessUniversalCheckoutComponentWithRedirectManagerEvents: String, CaseIterable {
    case onRetrieved, onError, onValid, onInvalid, onValidating
}
```

The only change would be at the callsite `.stringValue` -> `.rawValue`

Reply:
No, '@objc' enum raw type 'String' is not an integer type